src/java/picard/sam/markduplicates/util/OpticalDuplicateFinder.java

```diff
@@ -72,9 +72,12 @@ public OpticalDuplicateFinder(final String readNameRegex, final int opticalDupli
      * unmodified a copy of the list should be passed to this method.
      *
      * @param list a list of reads that are determined to be duplicates of one another
+     * @param keeper a single PhysicalLocation that is the one being kept as non-duplicate, and thus should never be
```

Comment:
I don't like `representative` because the name alone is not sufficient to convey that this _must_ be the records that is considered non-duplicate amongst the set.  How about `nonDuplicate` or `retained` instead?